OMPython/__init__.py

```diff
@@ -1773,6 +1773,19 @@ def getLinearMatrixValues(self,matrix):
             return tmpmatrix
         else:
             return np.zeros((0,0))
+        
+    def setCurrDir(mo, newPath):
+        path = newPath.replace('\\', '/')
+
+        if path[-1] == '/':
+            path = path[:-1]
+
+        if not os.path.exists(path):
+            raise Exception("Error: path does not exist")
+
+        mo.currDir = path
+        mo.requestApi('cd', entity=path)
+        return print(mo.getconn.sendExpression('cd()'))
```

Comment:
Why are you trying to call `cd` twice and why do you want to print it? Perhaps your intention is to call `getErroString` here. Also use `requestApi` instead of directly calling `sendExpression`.